src/Raven.Server/Documents/TombstoneCleaner.cs

```diff
@@ -184,6 +307,8 @@ internal TombstonesState GetState(bool addInfoForDebug = false)
                         }
                     }
                 }
+
+                CreateWarningIfThereAreBlockingTombstones(tombstoneCollections);
```

Comment:
When there are no disabled tasks that block the deletion of tombstones, you can dismiss the notification (need to add a test for that).

Reply:
done.